api/tests/unit/domain/services/assessment-service_test.js

```diff
@@ -464,10 +467,122 @@ describe('Unit | Domain | Services | assessment-service', () => {
     });
   });
 
+  describe('#findAssessmentByFilters', function() {
+
+    let sandbox;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
+      sandbox.stub(assessmentRepository, 'findByFilters').resolves([]);
+      sandbox.stub(certificationCourseRepository, 'get').resolves();
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should find all the assessment with corresponding filters', function() {
+      // given
+      const filters = { courseId: 'courseId' };
+
+      // when
+      const promise = service.findByFilters(filters);
+
+      // then
+      return promise.then(() => {
+        expect(assessmentRepository.findByFilters).to.have.been.called;
+        expect(assessmentRepository.findByFilters).to.have.been.calledWith({ courseId: 'courseId' });
+      });
+    });
+
+    context('when the assessment is a certification', () => {
+
```

Comment:
On pourrait remplacer, dans les it(...) suivants, 'assessment' par 'certification', pour mieux correspondre au cas de test